Viperinius.Plugin.SpotifyImport/Configuration/playlistConfig.js

```diff
@@ -32,21 +28,16 @@ function getUserSelectHtml(selectedUser) {
     return `<td class="detailTableBodyCell cellPlaylistUser"><select class="emby-select-withcolor emby-select" is="emby-select">${userOptionsHtml}</select></td>`;
 }
 
-function getIdTypeSelectHtml(selectedType) {
-    let typeOptionsHtml = '';
-    idTypes.forEach(type => {
-        typeOptionsHtml += `<option value="${type}" ${type === selectedType ? 'selected' : ''}>${type}</option>`;
-    });
+function getOnlyOwnHtml(onlyOwn) {
+    return `<td class="detailTableBodyCell cellPlaylistOnlyOwn"><input type="checkbox" ${onlyOwn ? 'checked' : ''}></td>`;
```

Comment:
Checkbox should use "Jellyfin style"
```suggestion
    return `<td class="detailTableBodyCell cellPlaylistOnlyOwn">
        <label class="emby-checkbox-label">
            <input type="checkbox" is="emby-checkbox" ${onlyOwn ? 'checked' : ''}/>
            <span></span>
        </label>
    </td>`;
```